ext/pdo_mysql/tests/pdo_mysql_quote_gbk_double_escape.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+PDO_MYSQL: Defining a connection charset in the DSN
+--EXTENSIONS--
+pdo_mysql
+--SKIPIF--
+<?php
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
+MySQLPDOTest::skip();
+?>
+--FILE--
+<?php
+    require_once __DIR__ . '/inc/mysql_pdo_test.inc';
```

Comment:
Unrelated, but what is the agreed upon policy when it comes to indentation in test files? Did we agree on any standard?

Reply:
No idea. From a quick glance it seems the pdo_mysql* tests use indented FILE sections while the bug tests don't...